docs/src/main/paradox/docs/releases/v1.10-release-notes.md

```diff
@@ -106,9 +107,9 @@ Previously deprecated storages can now be undeprecated.
 
 ### Resolvers
 
-#### Fetching the annotated original payload of a resolved resource.
+#### Fetching the annotated original payload of a Resolver resource.
```

Comment:
Maybe the wording could be improved but this change was about a resource fetched using resolvers not the resolver itself

Reply:
my bad I misunderstood the intent, will fix that in the doc and update the PR.